lib/driver.js

```diff
@@ -43,7 +43,6 @@ let serverCapabilities = {
 class IosDriver extends BaseDriver {
   resetIos () {
     this.appExt = ".app";
-    _.defaults(this.opts, serverCapabilities);
```

Comment:
i couldn't think of any reason we actually wanted these on `this.opts` other than to return them, which we do by extending down below. so removing for sanity. but it's possible these are needed somehow. cc @imurchie